projects/probational/cmd/probational.go

```diff
@@ -0,0 +1,249 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Moby is the type of a Moby config file
+// stolen from github.com/moby/tool/cmd/moby, perhaps we should make handling
+// this into a lib?
```

Comment:
we could probably vendor the type from the package itself, and then you could manually ignore the fields you aren't interested in (ex: `Trust`)